tests/unit/routers/contacts/test_api_post.py

```diff
@@ -120,12 +120,13 @@ def _change_primary_email(contact):
     _compare_written_contacts(saved_contacts[0], orig_sample, email_id)
 
 
-def test_create_with_non_json_is_error(client):
+def test_create_with_non_json_is_error(client, caplog):
     """When non-JSON is posted /ctms, a 422 is returned"""
     data = b"this is not JSON"
-    with capture_logs() as cap_logs:
+    with caplog.at_level(logging.INFO, logger="ctms.web"):
         resp = client.post("/ctms", content=data)
+
     assert resp.status_code == 422
     assert resp.json()["detail"][0]["msg"] == "JSON decode error"
-    assert len(cap_logs) == 1
-    assert "trace" not in cap_logs[0]
+    assert len(caplog.records) == 1
+    assert not hasattr(caplog.records[0], "trace")
```

Comment:
We can remove this assertion since we removed all the tracing stuff (https://github.com/mozilla-it/ctms-api/pull/944)